src/ucp/wireup/select.c

```diff
@@ -1570,28 +1593,22 @@ ucp_wireup_add_bw_lanes(const ucp_wireup_select_params_t *select_params,
                         ucp_wireup_select_context_t *select_ctx,
                         unsigned allow_extra_path)
 {
-    ucp_rsc_index_t skip_dev_index       = UCP_NULL_RESOURCE;
-    ucp_ep_h ep                          = select_params->ep;
-    ucp_context_h context                = ep->worker->context;
-    ucp_wireup_dev_usage_count dev_count = {};
+    ucp_rsc_index_t skip_dev_index        = UCP_NULL_RESOURCE;
```

Comment:
i think `skip_dev_index` must also be maintained at same level as `dev_count`.

Reply:
But unlike `dev_count`, we don't need to know about the `skip_dev_index` from one iteration to another. It will always be the same, right?

Reply:
yes we have one am lane and we need to exclude it once. with two rock nodes i see we that we fail to create the second path with run below:
- ` UCX_TLS=dc,cuda  UCX_RMA_BW_LANES_BATCH=1 ./rfs/bin/ucx_perftest -t tag_bw rock05`